### DISCUSSION TITLE
[Pulsar JVM Dashboard]  Zookeeper show No Data on column direct memory ?

### DISCUSSION THREAD

**Discussion by lthedung** [apache/pulsar#18382] [Q&A]
**Describe the bug**

To monitor a Pulsar cluster, I used grafana and in the Pulsar JVM dashboard, Zookeeper Direct Memory is show No Data. While other dashboards have data such as Bookie, Broker, Recovery.

I want to know why Zookeeper Metrics has no data for direct memory. I researched the document but did not find information about zookeeper metrics. Can you help me?

**Zookeeper**
![image](https://user-images.githubusercontent.com/56428954/200501163-e78fe91b-0539-4b8b-a445-a6a459cce168.png)

**Broker**
![image](https://user-images.githubusercontent.com/56428954/200501097-99692ea4-cba6-4b06-8264-94af463a14ed.png)

**Recovery**
![image](https://user-images.githubusercontent.com/56428954/200501235-b37c8d69-8ad0-4026-81d3-c8bcb9d33dbe.png)

**Grafana Overview**
![image](https://user-images.githubusercontent.com/56428954/200501319-143203e6-f690-4483-940b-11dc345fa2c7.png)

**Comment by eolivelli** [comment]
which version of Pulsar ?
which dashboards are you using ?

**Reply by lthedung** [reply]
Pulsar version 2.9.2
Dashboard grafana
